### ISSUE TITLE
Links in the Repo leading to no existing files

### ISSUE THREAD

**Issue by kevin27luedemann** [HPC-certification-forum/www#1]
Trying to run the webpage locally leads into the problem that the two files:

* data/examiner.php
* data/verifier.php

cannot be found. Therefore, running `jekyll serve` stops after reaching those files.
A hot fix is to simply delete the files, but a more permanent solution is required.
Are these files part of a different repository?

**Comment by JulianKunkel** [comment]
indeed, they are part of the examination repo. your temp fix is fine but agreed, a better structure can be thought of
